src/main/kotlin/no/nav/familie/ks/sak/kjerne/behandling/steg/vilkårsvurdering/VilkårsvurderingService.kt

```diff
@@ -30,6 +32,7 @@ class VilkårsvurderingService(
     private val personopplysningGrunnlagService: PersonopplysningGrunnlagService,
     private val sanityService: SanityService,
     private val personidentService: PersonidentService,
+    private val unleashService: UnleashService,
```

Comment:
Vi har ellers i løsningen lagt opp til at vi bruker `UnleashNextMedContextService` fremfor `UnleashService` for toggelen `STØTTER_LOVENDRING_2025`. Da kan vi åpne opp for at bare Dorota og Anna f.eks har tilgang til ny funksjonalitet.